versioned_docs/version-2.8/api/previous-rancher-api-versions/previous-rancher-api-versions.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Previous API Guide
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/api/previous-rancher-api-versions"/>
+</head>
+
+## How to use the API
+
+The previous Rancher API has its own user interface accessible from a web browser.  This is an easy way to see resources, perform actions, and see the equivalent cURL or HTTP request & response.  To access it:
```

Comment:
We should probably mention v2.7 explicitly. When we cut a new version of the docs for v2.9, v2.10, etc. what "previous" refers to here will be at more and more of a remove from the current latest

Reply:
Updated the phrasing with the version being mentioned now, let me know if it sounds alright. Also thanks for the review Marty!